staging_service/app.py

```diff
@@ -241,15 +239,6 @@ async def get_jgi_metadata(request: web.Request):
     return web.json_response(await read_metadata_for(path))
 
 
-@routes.get("/impoter-defaults/{path:.*}")
```

Comment:
I assume this route is unused in KBase?

Reply:
Yea and the function it went to was a broken empty function actually, so i deleted it too